### DISCUSSION TITLE
what happened to kubernetes support?

### DISCUSSION THREAD

**Discussion by Hr46ph** [wolveix/satisfactory-server#236] [General]
In the readme it mentions 
"If you are running a [Kubernetes](https://kubernetes.io/) cluster, we do have a [service.yaml](https://github.com/wolveix/satisfactory-server/cluster/service.yaml) and [statefulset.yaml](https://github.com/wolveix/satisfactory-server/cluster/statefulset.yaml) available under the [cluster](https://github.com/wolveix/satisfactory-server/cluster) directory of this repo, along with an example [values.yaml](https://github.com/wolveix/satisfactory-server/cluster/values.yaml) file."

But the links are dead. The Helm repo points to a project that has been archived and unmaintained.

Besides, I prefer manifests over Helm.

Are you still maintaining the project?

Thanks! 😄

**Comment by wolveix** [comment]
Hey! The links are incorrect, but you can see the files still clearly exist under the `cluster` directory.

With regards to the Helm project, that is not maintained by me, so I can't speak to it. It's very easy to use the files under the `cluster` directory and be up and running in no time :)

**Reply by Hr46ph** [reply]
Thanks! Been busy with some other stuff as well and last week finally found the time to get into getting this up and running and moving my server from a single docker host to a 5 node Kubernetes cluster.

Had to make some minor changes for my storage provider and network setup. It all works really well.

Cheers!
